MVP_Clicker/Assets/Project/Scripts/Game/Areas/Achievement/Types/MonsterKillingAchievementModel.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using Project.Scripts.Game.Areas.Achievement.Data;
+using Project.Scripts.Game.Areas.Monster.Model;
+
+namespace Project.Scripts.Game.Areas.Achievement.MonsterKillingAchievement.Model
+{
+    public class MonsterKillingAchievementModel : IAchievementModel, IDisposable
+    {
+        public event Action Updated;
+        public event Action AchievementCompleted;
+
+        private readonly IMonsterModel _monster;
+        public IAchievementData _data;
+
+        public int RequiredPointsToComplete { get; }
+
+        public int CurrentPoints
+        {
+            get => _data.CurrentPoints;
+            private set => _data.CurrentPoints = value;
+        }
+
+        public string Description { get; }
+        public string Name { get; }
+
+        public MonsterKillingAchievementModel(IMonsterModel monster, IAchievementConfig config, IAchievementData data)
+        {
+            _monster = monster;
+            _data = data;
+            AddListeners();
+            RequiredPointsToComplete = config.RequiredPointsToComplete;
+            Description = config.Description;
+            Name = config.Name;
+        }
+
+        private void GetPoint()
+        {
+            if (CurrentPoints + 1 >= RequiredPointsToComplete)
```

Comment:
don't do calculation inside if condition. It's difficult to manage and understand it. Move it to own field or just move line 43 above (I mean the thing that you doing with CurrentPoints++)